pom.xml

```diff
@@ -240,9 +241,9 @@
 			<version>${slf4j.version}</version>
 		</dependency>
 		<dependency>
-			<groupId>org.slf4j</groupId>
-			<artifactId>slf4j-log4j12</artifactId>
-			<version>${slf4j.version}</version>
+			<groupId>org.apache.logging.log4j</groupId>
+			<artifactId>log4j-slf4j-impl</artifactId>
```

Comment:
org.slf4j:slf4j-log4j12 seems almost dead. Activity is now on log4j-slf4j-impl

Reply:
with 2.16 we are on the safe side, right?

Reply:
it is the latest release as of today which is not impacted by log4jshell CVE. (this version has even completely removed the feature of jndi lookup)